dev/com.ibm.ws.security.social/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -188,6 +188,9 @@
             required="true" type="String" ibm:type="password" />
         <AD id="authorizationEndpoint" name="%authorizationEndpoint" description="%authorizationEndpoint.desc"
             required="false" type="String" default="https://github.com/login/oauth/authorize"/>
+        <AD id="introspectionTokenTypeHint" name="internal" description="internal use only" type="String" required="false" default="access_token">
```

Comment:
Looks like this was actually put under the `com.ibm.ws.security.social.github.metatype` metatype, not the `oauth2Login` metatype. We'll have to come back and update this.